content/management/changelog/index.mdx

```diff
@@ -21,14 +21,29 @@ Since the migration is still ongoing, different parts will be successively moved
 - The `agent_status_changed` webhook was replaced with [`routing_status_set`](/management/configuration-api/v3.2/#routing_status_set).
 - There's a new field, `license_id`, in all [webhooks](/management/configuration-api/v3.2/#webhooks).
 - From now on, it's possible to define properties in [2 new locations](/management/configuration-api/v3.2/#property-locations): `license` and `group`.
-- There are 6 new methods: [**Set License Properties**](/management/configuration-api/v3.2/#set-license-properties), [**Get License Properties**](/management/configuration-api/v3.2/#get-license-properties), [**Delete License Properties**](/management/configuration-api/v3.2/#delete-license-properties), [**Set Group Properties**](/management/configuration-api/v3.2/#set-group-properties), [**Get Group Properties**](/management/configuration-api/v3.2/#get-group-properties), [**Delete Group Properties**](/management/configuration-api/v3.2/#delete-group-properties).
+- There are 6 new methods: [**Update License Properties**](/management/configuration-api/v3.2/#update-license-properties), [**List License Properties**](/management/configuration-api/v3.2/#list-license-properties), [**Delete License Properties**](/management/configuration-api/v3.2/#delete-license-properties), [**Update Group Properties**](/management/configuration-api/v3.2/#update-group-properties), [**List Group Properties**](/management/configuration-api/v3.2/#list-group-properties), [**Delete Group Properties**](/management/configuration-api/v3.2/#delete-group-properties).
```

Comment:
I don't know but this changelog shows changes from v3.1 to v3.2. And from this perspective we have added new methods list/update. Wdyt? I don't know how to show changes in v3.2 (and historically it will be useless and confusing after v3.2 gets stable).

Reply:
That's true, it will be confusing in the future. It might be confusing for those who currently use 3.2, as they might not notice in the Changelog that we renamed the methods. But we also have other channels of communicating these changes, and not many people use this version now.

Reply:
I was also wondering if it's a good idea to add the newest changes at the bottom of the list in Changelogs. It would require some work to reverse the order, so maybe instead, we could start adding months annotations, for version 3.3

---- October 2020 -----
- change 1 
- change 2

----- November 2020 ----
- change 1
-change 2

Reply:
Ok, that's an idea, we can talk about it more